src/pages/tutorials/turbonomic/kubeturbo/index.mdx

```diff
@@ -193,7 +193,7 @@ The steps in this lab should be performed on the **manage-to** cluster:
 
 1. Then set a cluster name for the `targetName`, e.g.
 
-   - `manage-to-kubeturbo-cluster`.
+   - `application-kubeturbo-cluster`.
```

Comment:
Does this cluster name reference appear anywhere else in the turbonomic labs and need updating?

Reply:
No it doesn't. This one instance is a more descriptive name (in the context of the Turbo lab) set in Turbo for referencing the application cluster.

Reply:
Cool. Will wait for @deepblue2k to push up any remaining diagram changes and then merge.